Tests/FuzzilliTests/VariableMapTest.swift

```diff
@@ -35,7 +35,8 @@ class VariableMapTests: XCTestCase {
         XCTAssert(m.contains(v(1)) && m[v(1)] == 1)
         
         m.remove(v(1))
-        XCTAssert(!m.contains(v(1)) && m[v(1)] == nil)
+
+        XCTAssert(!m.contains(v(42)) && m[v(42)] == nil)
```

Comment:
This was correct the way it was before, i.e. `XCTAssert(!m.contains(v(1)) && m[v(1)] == nil)`

Reply:
Changes reverted to the original implementation